scripts/similarity_query.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import sys
+
+from bugbug import bugzilla, similarity
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--algorithm",
+        help="Similarity algorithm to use",
+        choices=similarity.model_name_to_class.keys(),
+    )
+    parser.add_argument("--bug_id", help="Bug to test")
+    return parser.parse_args(args)
+
+
+def main(args):
+    model = similarity.model_name_to_class[args.algorithm]().load(
```

Comment:
You made `load` a static method, so you don't need to instantiate an object in order to call it

Reply:
Ahh, sorry, missed this.